com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -847,5 +847,32 @@ internal bool ValidateRendererData(int index)
             if (index == -1) index = m_DefaultRendererIndex;
             return index < m_RendererDataList.Length ? m_RendererDataList[index] != null : false;
         }
+
+        /// <summary>
+        /// Check the list of Renderers against all Graphics APIs the player is built with.
+        /// </summary>
+        /// <returns></returns>
+        internal bool ValidateRendererGraphicsAPIs(out string unsupportedGraphicsApisMessage)
```

Comment:
Since the only use of this is in `UniversalRenderPipelineAssetEditor` and it looks very UI-specific given that it constructs a string for display, should this maybe just live there instead? Then we don't have to make it `internal` either. You can use `rendererIndexList` and `GetRenderer(int)` to access the renderers.

Reply:
I moved `ValidateRenderGraphicsAPIs()` in `UniversalRenderPipelineAssetEditor`.